collections/requirements.yml

```diff
@@ -1,2 +1,3 @@
+---
 collections:
-  - name: "chrismeyersfsu.test_things"
```

Comment:
Will break some tower qa tests that we have that depend on this. I am all for renaming the collection they depend on. In other words, if we merge this, a follow up PR to the QE tests will be required.

Reply:
:+1: got it, I can open one so we can merge them close to the same time and avoid frustrations.